{
    "checksum": "2b5229ca786bb0c3480a56890fcfbc5f01f975898520021e0d8a28dd8ea70d69",
    "file": "qt-Fix-exception-with-Android-5.x.patch",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-diff"
    },
    "number_of_duplicates": 2,
    "package": "ausweisapp2",
    "path": "ausweisapp2/1.22.2-1/libs/patches/qt-Fix-exception-with-Android-5.x.patch",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/ausweisapp2/1.22.2-1/",
        "metric": {
            "size": 22820
        },
        "pts_link": "https://tracker.debian.org/pkg/ausweisapp2",
        "sloc": [
            [
                "cpp",
                87286
            ],
            [
                "python",
                1371
            ],
            [
                "xml",
                1044
            ],
            [
                "java",
                637
            ],
            [
                "sh",
                147
            ],
            [
                "makefile",
                5
            ]
        ],
        "suites": [
            "bookworm",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/debian/ausweisapp2",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/a/ausweisapp2/1.22.2-1/libs/patches/qt-Fix-exception-with-Android-5.x.patch",
    "stat": {
        "perms": "rw-r--r--",
        "size": 2246,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.22.2-1"
}